RC Multi-factor authentication: updated language to fit new UI (#2112)
RedisLabs/redislabs-docs: content/rc/security/multi-factor-authentication.md

1. Log in to your account.
2. From the Redis Enterprise Cloud menu, click on your name to enter the **User Profile** view.
3. Under your user profile, locate **Multi-factor authentication**.

    {{<image filename="images/rc/multi-factor-authentication-user-profile.png" alt="Multi-factor authentication is located on the user profile page." >}}{{< /image >}}

> 4. Enter your mobile phone number, and click **Send code**.

Reviewer: ```suggestion
4. Enter your mobile phone number in the **Text message** box and then select **Send code**.
```

> 5. You will receive a confirmation code sent by text message. Enter the code and click **Verify**.

Reviewer: Does the admin console show a prompt where you are supposed to enter the code? If so, you might want to clarify that here.
```suggestion
5. You will receive a confirmation code sent by text message. Enter the code and select **Verify**.
```

Reviewer: @Sawdustie, I changed this back to unresolved because I'm not sure you saw my question about clarifying that you enter the code into a prompt in the admin console (if this statement is true).

Author: You're right I did not see it, sorry. The admin console does show a prompt where the code should be entered, I will add that.

To change the mobile phone number used for MFA:

1. Navigate to the **Multi-Factor Authentication** view.

> 1. Click **Change number**.

Reviewer: ```suggestion
1. Select **Change number**.
```

1. Install an authenticator app on your mobile phone.
1. Add Redis Cloud to the app:
    1. From the **User Profile** view in your Redis Cloud account, locate **Multi-Factor Authentication**.

>     1. Click **Connect** for the authenticator app. A QR code will appear on screen requesting verification.

Reviewer: ```suggestion
    1. Select **Connect** for the authenticator app. A QR code will appear on screen requesting verification.
```

    1. Scan the QR code using your phone's authenticator app.
    1. Enter the code generated by your authenticator app to verify the setup.

You can now use either a text message code or an authenticator app code as your second factor when logging in.

#### Deactivating MFA

> You can deactivate MFA for your user account. To deactivate MFA, go to your profile, locate **Multi-Factor Authentication**, and click **Deactivate**.

Reviewer: ```suggestion
You can deactivate MFA for your user account. To deactivate MFA, go to your profile, locate **Multi-Factor Authentication**, and select **Deactivate**.
```
